packages/sanitize/src/index.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Strip HTML tags.
+ *
+ * @param {string} text Text to have the HTML tags striped out of.
+ *
+ * @return  Stripped text.
+ */
+export function stripTags( text ) {
+	text = text || '';
+
+	return text
+		.replace( /<!--[\s\S]*?(-->|$)/g, '' )
+		.replace( /<(script|style)[^>]*>[\s\S]*?(<\/\1>|$)/ig, '' )
+		.replace( /<\/?[a-z][\s\S]*?(>|$)/ig, '' );
+}
+
+/**
+ * Strip HTML tags and convert HTML entities.
+ *
+ * @param {string} text Text to strip tags and convert HTML entities.
+ *
+ * @return Sanitized text. False on failure.
+ */
+export function sanitizeText( text ) {
```

Comment:
This needs a better name and description.  `sanitizeText` - in what way, against what? `convert HTML entities` - into what?

If I read this correctly, I think the answers are "remove tags and encode HTML entities".  I'm having a hard time thinking of a good function name, though.  Maybe we should just create the `encodeHTMLEntities` first.